app/components/Bookings/Details/header.js

```diff
@@ -286,6 +288,10 @@ const Header = ({
                       <EliteOfficerName staffId={inmateData.getIn(['keyworker', 'staffId'])} />
                     )}
                   </strong>
+                  <span className="label">Last session</span>
+                  <strong>
+                    {lastKeyWorkerSessionDate ? <FormattedDate value={lastKeyWorkerSessionDate} /> : '--'}
+                  </strong>
```

Comment:
Should this be added to the mobile version as well? I'm guessing that's what circle is complaining about.

Reply:
Tests were failing because the button was technically visible but hidden behind the mobile toolbar.
![image](https://user-images.githubusercontent.com/1067537/74452067-77bae300-4e78-11ea-8297-93b02e111173.png)

Hopefully fixed the tests by scrolling down the page before that button is clicked.